FWCore/Framework/src/Schedule.cc

```diff
@@ -1052,7 +1065,7 @@ namespace edm {
   }
                           
   void
-  Schedule::getTriggerTimingReport(TriggerTimingReport& rep) const {
+  Schedule::getTriggerTimingReport(TriggerTimingReport& rep) {
```

Comment:
@wmtan I looked at this can remain ```const``` if we simply change ```SystemTimKeeper::fillTriggerTimingReport``` to be a ```const``` function. I already tested changing that member function to be ```const``` and everything compiled.